src/screens/Subscriptions/AfterUnsubscribe.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { createRef } from 'react';
+import { Wrapper, Header, BodyCopy } from './Subscriptions.style';
+import { EventAction, EventCategory } from 'components/Analytics';
+import ExternalLink from 'components/ExternalLink';
+import { LargeFilledButton } from 'components/ButtonSystem';
+
+const AfterUnsubscribe = () => {
+  const params = new URLSearchParams(window.location.search);
+  const email = params.get('email') || '';
+  const formRef = createRef<HTMLFormElement>();
+
+  const formAction = getFormActionLink(email);
```

Comment:
This odd code to get the form action link from campaign monitor and add it to the form before submitting it is modeled off of our existing code for daily download signup (https://github.com/covid-projections/covid-projections/blob/develop/src/components/EmailAlertsForm/utils.ts#L37)...